packages/edit-site/src/components/editor/global-styles-renderer.js

```diff
@@ -71,7 +86,7 @@ function getBlockPresetClasses( blockSelector, blockPresets = {} ) {
 					const value = preset[ valueKey ];
 					const classSelectorToUse = `.has-${ slug }-${ classSuffix }`;
 					const selectorToUse = `${ blockSelector }${ classSelectorToUse }`;
-					declarations += `${ selectorToUse } {${ propertyName }: ${ value };}`;
+					declarations += `${ selectorToUse }{${ propertyName }: ${ value } !important;}`;
```

Comment:
Extracted PR at https://github.com/WordPress/gutenberg/pull/31204 (this is a bug)